site/source/tutorials/1050_development_environment/index.md

```diff
@@ -161,4 +168,11 @@ Chrome Accessibility Developer Tools does not run an audit against your code, or
 
 Dojo 2 can be used out of the box with any code editor, but good TypeScript support will provide integrated code help and intellisense. Together with robust in-browser debugging, this should set you up with a streamlined process for editing, troubleshooting, and committing code.
 
+Moving forward, you can:
+
+* Explore other [TypeScript compatible editors](https://github.com/Microsoft/TypeScript/wiki/TypeScript-Editor-Support)
+* Use _code editor snippets_ to quickly generate code snippets from simple shortcuts (see docs for [VS Code](https://code.visualstudio.com/docs/editor/userdefinedsnippets), [Sublime Text](http://docs.sublimetext.info/en/latest/extensibility/snippets.html) & [Atom](http://flight-manual.atom.io/using-atom/sections/snippets/))
+* Familiarise yourself with browser developer tooling such as the [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/), [Firefox Developer Tools](https://developer.mozilla.org/son/docs/Tools), [Safari Web Inspector](https://developer.apple.com/library/content/documentation/NetworkingInternetWeb/Conceptual/Web_Inspector_Tutorial/EnableWebInspector/EnableWebInspector.html) or the [Edge F12 Developer Tools](https://docs.microsoft.com/en-us/microsoft-edge/f12-devtools-guide). Understanding developer tooling will help you effectively edit and debug Dojo 2 web applications
```

Comment:
We've decided to use US English for consistency in the docs, so it should be Familiarize

Reply:
Got it, done!